frameworks/hdfs/tests/test_upgrade_downgrade.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import sdk_versions
+from tests.config import *
+
+@pytest.mark.upgrade
+@pytest.mark.sanity
+def test_upgrade():
+    options = {
+        "service": {
+            "name": "beta-{}".format(PACKAGE_NAME),
```

Comment:
I'll rerun it to confirm but I remember when I actually ran the test without `beta-{}` there ended up being a discrepancy between service names so the polling for deployment and recovery would be polling using the wrong name and as a result would timeout.